File: common/src/main/java/com/tnt_man_inc/jarm_3/blocks/BaseBlockItem.java
package com.tnt_man_inc.jarm_3.blocks;


import com.tnt_man_inc.jarm_3.Jarm;
import net.minecraft.world.item.BlockItem;
import net.minecraft.world.level.block.Block;

public class BaseBlockItem extends BlockItem {
    public BaseBlockItem(Block block, Properties properties) {
        super(block, properties.tab(Jarm.JARM_TAB));
    }
}

File: common/src/main/kotlin/com/tnt_man_inc/jarm_3/blocks/BaseBlockItem.kt
package com.tnt_man_inc.jarm_3.blocks

import com.tnt_man_inc.jarm_3.Jarm
import com.tnt_man_inc.jarm_3.items.NISGDApple
import net.minecraft.world.item.BlockItem
import net.minecraft.world.level.block.Block
import java.lang.AssertionError

class BaseBlockItem(block: Block?, properties: Properties) : BlockItem(block, properties.tab(Jarm.JARM_TAB))